src/components/digits.js

```diff
@@ -1,25 +1,26 @@
 import React from 'react';
 
-const Digits = () => (
+// eslint-disable-next-line react/prop-types
+const Digits = ({ handleEvent }) => (
   <div className="calculator-lower">
     <div className="calculator-number">
-      <button type="button" alt="">7</button>
-      <button type="button" alt="">8</button>
-      <button type="button" alt="">9</button>
+      <button type="button" alt="" onClick={handleEvent} value="7">7</button>
+      <button type="button" alt="" onClick={handleEvent} value="8">8</button>
+      <button type="button" alt="" onClick={handleEvent} value="9">9</button>
     </div>
```

Comment:
- [ ] Kindly remove the `eslint-disable` statement in this file. `Eslint` helps us to write clean, readable and industry-standard JavaScript codes. Using `eslint-disable` statements defeats this purpose. Kindly remove it. 👍 💻